CONTRIBUTING.md

```diff
@@ -21,7 +33,7 @@ Yarn is a package manager for your code, similar to npm. While you can use npm t
 3. In your terminal, navigate to the directory you cloned Spectral into (check that you are on the `develop` branch).
 4. Install the dependencies: `yarn`
 5. Build Spectral: `yarn build`
-6. Run Spectral from your local installation: `yarn cli lint [openapi_spec_file]`
+6. Run Spectral from your local installation: `yarn cli cli lint [openapi_spec_file]`
```

Comment:
Is this correct?

Reply:
How about https://github.com/stoplightio/spectral/commit/82688685b9925c0c7041bc3efd078f901b70e256